spa/src/utils/dynamicTableHelper.tsx

```diff
@@ -0,0 +1,149 @@
+import Avatar from "@atlaskit/avatar";
+import styled from "@emotion/styled";
+import Badge from "@atlaskit/badge";
+import { token } from "@atlaskit/tokens";
+import Lozenge from "@atlaskit/lozenge";
+import { SuccessfulConnection } from "../rest-interfaces";
+import { ThemeAppearance } from "@atlaskit/lozenge/dist/types/Lozenge";
+
+type Row = {
```

Comment:
The indentation on all the other files is different from this one. Surprised the linter hasn't complained 🤔

Reply:
Same here, will be done in [ARC-2467](https://softwareteams.atlassian.net/browse/ARC-2467)

Reply:
done